miner/scroll_worker.go

```diff
@@ -579,11 +583,19 @@ func (w *worker) startNewPipeline(timestamp int64) {
 			return
 		}
 	}
+
+	// pipelineCCC was nil, so the block was built for RPC purposes only. Stop the pipeline immediately
+	// and update the pending block.
+	if pipelineCCC == nil {
+		w.currentPipeline.Stop()
```

Comment:
Why use Stop() here instead of Release()?

Reply:
`Release()` might consume the result that pipeline returns. We still want worker to be able to read the result and process.